model/common/tests/common/interpolation/stencil_tests/test_mo_intp_rbf_rbf_vec_interpol_vertex.py

```diff
@@ -33,7 +33,7 @@ def reference(
         ptr_coeff_1: np.ndarray,
         ptr_coeff_2: np.ndarray,
         **kwargs: Any,
-    ) -> dict[str, np.ndarray]:
+    ) -> dict:
```

Comment:
why this change?

Reply:
otherwise you get this: 
```
model/common/tests/common/interpolation/stencil_tests/test_mo_intp_rbf_rbf_vec_interpol_vertex.py:30:5: error: Signature of "reference" incompatible with supertype "StencilTest"  [override]
```